components/base/DHero.vue

```diff
@@ -39,3 +44,31 @@ export default {
   },
 };
 </script>
+
+<style lang="scss" scoped>
+.header-image {
```

Comment:
it seems most of this and `.header-image-alt` are the same.  Maybe make three classes `header-image`, which is the common things, then `image-home` and `image-alt` or something for the actual images?